src/ui_fsmenu/options.cc

```diff
@@ -623,12 +623,13 @@ void Options::update_language_stats() {
 			if (language_entries_.count(parts[0]) == 1) {
 				locale = parts[0];
 			} else {
-				split(parts, parts[0], {'@'});
-				if (language_entries_.count(parts[0]) == 1) {
+				std::vector<std::string> parts2;
+				split(parts2, parts[0], {'@'});
+				if (language_entries_.count(parts2[0]) == 1) {
 					locale = parts[0];
 				} else {
-					split(parts, parts[0], {'_'});
-					if (language_entries_.count(parts[0]) == 1) {
+					split(parts, parts2[0], {'_'});
+					if (language_entries_.count(parts2[0]) == 1) {
```

Comment:
Shouldn't this be `parts[0]`?

Reply:
Yes you are correct, I cannot get a ref for i18n::get_locale() and its possible values.